### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by storyworks**
https://3.basecamp.com/3105655/buckets/23904268/todolists/4127141243

### CODE REVIEW

**Review by tataihono — CHANGES_REQUESTED** [review]

File: libs/shared/ui/src/components/Typography/Typography.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import Typography from './Typography'
+
+describe('Typography', () => {
+  it('should render successfully', () => {
+    const { getByRole } = render(
+      <Typography
+        __typename="TypographyBlock"
+        id="heading3"
+        parentBlockId="question"
+        content={'Hello World!'}
```
Comment (tataihono) [inline libs/shared/ui/src/components/Typography/Typography.spec.tsx:18]:
```suggestion
        content="Hello World!"
```

File: libs/shared/ui/src/components/Typography/Typography.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        variant={TypographyVariant.h3}
+        color={TypographyColor.primary}
+        align={TypographyAlign.left}
+      />
+    )
+    expect(
+      getByRole('heading', { name: 'Hello World!', level: 3 })
+    ).toBeTruthy()
```
Comment (tataihono) [inline libs/shared/ui/src/components/Typography/Typography.spec.tsx:26]:
```suggestion
    ).toBeInTheDocument()
```

File: libs/shared/ui/src/components/Typography/Typography.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { ReactElement } from 'react'
+import {
+  Typography as MuiTypography,
+  TypographyProps as MuiTypographyProps
+} from '@mui/material'
+import { TypographyFields } from './__generated__/TypographyFields'
+
+export interface TypographyProps
```
Comment (tataihono) [inline libs/shared/ui/src/components/Typography/Typography.tsx:8]: These props should just be from the API not from Mui Typography.

**Review by tataihono — CHANGES_REQUESTED** [review]
please move typography block component back into the journeys project

**Review by tataihono — CHANGES_REQUESTED** [review]
I would have expected to see pages/[journeyId].tsx to be updated to include the new fragment. Can you add that in then this should be done :)

**Comment by tataihono** [comment]
can you generate types for your query adjustment `nx codegen journeys`

**Comment by storyworks** [comment]
> can you generate types for your query adjustment `nx codegen journeys`

I have run that? This was the change
image

**Review by tataihono — APPROVED** [review]

**Pull request merged**